README.md

```diff
@@ -91,11 +91,11 @@ If using PostgreSQL, to administer the database manually (not essential, but use
 We provide project files to help you build and browse code using IntelliJ IDEA.
 If you wish to take advantage of this convenience, you should also install IntelliJ.
 
-
 Then:
 * Pull the repository
-* Set variables. Duplicate `/resources/variables-template.sh`, and overwrite with values suitable for your environment. Copy it to a convenient location and set an environmental variable VARIABLES_FILE to point to it using the full path. We recommend you set that up in .bashrc so that it is always definted correctly.
-* The favoured (and our current) approach is to use 'in memory' persistence, specified in `node.properties` in the working folder. However, postgres is an option. If using postgres, setup and run the db. Run `/bin/db/setup.sh`
+* Build the project using `mvn package -f code/core/pom.xml`
```

Comment:
Hi Abdel, thanks for the feedback. I don't know the answer to the readme / mvn package step. I think it depends on how it is being run and deployed. I will have to have a chat with Gideon about this and we'll also talk about the more confusing bits of the intro. Perhaps I will add some info about the update() call and so on. Will get back to you.